src/main/java/com/ft/methodearticlemapper/transformation/EomFileProcessor.java

```diff
@@ -219,6 +224,35 @@ private Syndication getSyndication(final XPath xpath, final Document attributesD
         }
     }
 
+    private ContentPackage getContentPackage(final XPath xpath,
+                                             final Document attributesDocument,
+                                             final Document valueDocument) throws TransformerException, XPathExpressionException {
+
+        final String isContentPackage = xpath.evaluate("/ObjectMetadata/OutputChannels/DIFTcom/isContentPackage", attributesDocument);
+        if (!Boolean.TRUE.toString().equalsIgnoreCase(isContentPackage)) {
+            return null;
+        }
+
+        final Node descriptionNode = (Node) xpath.evaluate("/doc/lead/lead-components/content-package/content-package-description", valueDocument, XPathConstants.NODE);
+        if (descriptionNode == null) {
+            return null;
+        }
+
+        final String description = getNodeAsString(descriptionNode.getFirstChild());
+        if (StringUtils.isBlank(description)) {
+            return null;
+        }
+
+        final String linkHref = xpath.evaluate("/doc/lead/lead-components/content-package/content-package-link/a/@href", valueDocument);
+        final String linkId = StringUtils.substringAfter(linkHref, "uuid=");
+
+        try {
+            return new ContentPackage(description.trim(), UUID.fromString(linkId.trim()).toString());
+        } catch (IllegalArgumentException e) {
+            return null;
```

Comment:
Done. Also added some warnings in case the description is not found and removed a duplicate logger (like a good boy scout)